docker/README.md

```diff
@@ -10,7 +10,7 @@ https://docs.nvidia.com/datacenter/cloud-native/container-toolkit/install-guide.
 
 ```
 # Build:
-DOCKER_BUILDKIT=1 docker build --build-arg BASE={PyTorch Base Container Version} -f docker/Dockerfile -t torch_tensorrt1.0:latest .
+DOCKER_BUILDKIT=1 docker build --build-arg BASE={TensorRT Base Container Version} -f docker/Dockerfile -t torch_tensorrt1.0:latest .
```

Comment:
Should we use NGC CUDA container instead?
CI will be based on NGC CUDA container.

This way we have:
NGC branch from NGC PyTorch container
Master branch from TensorRT container
CI from NGC CUDA container

Reply:
We need tensorrt for external use since there's no curl able public trt package